packages/protocol/scripts/foundry/create_and_migrate_anvil_devchain.sh

```diff
@@ -60,6 +60,25 @@ $NON_INTERACTIVE \
 $LIBRARY_FLAGS \
 --rpc-url $ANVIL_RPC_URL || echo "Migration script failed"
 
+# Set the supply for Celo:
+# This supply is just an approximation, but it's importat that this number is non-zero.
+
+# get celo contract address
+CELO_ADDRESS=`cast call $REGISTRY_ADDRESS "getAddressForStringOrDie(string calldata identifier) returns (address)" "CeloToken" --rpc-url $ANVIL_RPC_URL`
```

Comment:
Nit: 

```suggestion
# Get CeloToken contract address
CELO_ADDRESS=$(
  cast call \
  $REGISTRY_ADDRESS \
  "getAddressForStringOrDie(string calldata identifier)(address)" \
  "CeloToken" \
  --rpc-url $ANVIL_RPC_URL
)
```

To be consistent with how it's done here: 

https://github.com/celo-org/celo-monorepo/blob/e6dcb576481a9232713b6fb9354a7bdf00e82f33/packages/protocol/scripts/foundry/create_and_migrate_anvil_l2_devchain.sh#L39-L45